src/parser/es7.js

```diff
@@ -1,6 +1,7 @@
 import { parse } from '@babel/parser';
+import { getContent } from '../utils/file';
 
-export default function parseES7(content) {
+export function parseES7Content(content) {
```

Comment:
Had to extract that in a separated function because this is needed in some specials.